### ISSUE TITLE
How should set validation_batch_size??

### ISSUE THREAD

**Issue by jungsdao** [ACEsuit/mace#162]
Hello, 
I'd like to ask if there are some heuristics to set `validation_batch_size` ?
Normally validation set size is way smaller than training set. So having the same value for `batch_size` might be sensible. 
Could you give me some advice on this?
Thank you in advance!
